engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ExternalDeploymentId.scala

```diff
@@ -0,0 +1,12 @@
+package pl.touk.nussknacker.engine.api.deployment
+
+import io.circe.generic.extras.semiauto.{deriveUnwrappedDecoder, deriveUnwrappedEncoder}
+import io.circe.{Decoder, Encoder}
+
+//id generated by external system - e.g. Flink JobID
+final case class ExternalDeploymentId(value: String) extends AnyVal
```

Comment:
:+1: for introducing new name to clarify that it is not and id passed from ui. But maybe instead of external we could name it engine? Because external looks like it is defined by some other system

Reply:
For me it's external - outside NK?

Reply:
If you will have embedded engine that will generate own id would it be still external? ... but it is only suggestion so you can skip it if you want.